components/mitsubishi_uart/muart_packet.h

```diff
@@ -132,29 +131,15 @@ class ExtendedConnectResponsePacket : public Packet {
   float getMinAutoSetpoint() const { return ((int) pkt_.getPayloadByte(14) - 128) / 2.0f; }
   float getMaxAutoSetpoint() const { return ((int) pkt_.getPayloadByte(15) - 128) / 2.0f; }
 
+  // Things that have to exist, but we don't know where yet.
+  bool supportsHVaneSwing() const { return false; }
```

Comment:
Default this to true so that systems that *do* support HVane aren't stuck without it (and anyone without it can just disable it the way it's being disabled now)

Reply:
n.b. this code actually doesn't do anything yet, and likely won't for a while. Likewise, we don't actually have any overrides for climate card vane things, so no override (either to enable or disable).

Point taken though and I will tie it to normal vane control for now.

Reply:
I still think this needs to default to `true`.

I don't actually have a lot of knowledge of the gamut of equipment options, but tying it to the vertical would still disable it for anyone with a horizontal-only swing device.

There is a disable override available via ESPHome config by setting disabled_by_default to true:
``` yaml
mitsubishi_uart:
  heatpump_uart: hp_uart
  thermostat_uart: tstat_uart
  temperature_sources:
    - fake_temp
    - home_assistant_temperature
  selects:
    horizontal_vane_position_select:
      name: Horizontal Vane Position
      disabled_by_default: true
      id: mitsubishi_uart_horizontalvanepositionselect
      entity_category: config
      icon: mdi:arrow-expand-horizontal
```
or just by disabling it via Home Assistant directly.

I *know* this code doesn't do anything yet, but it should still be correct so that when it does get used we don't have to come back and fix this.

Reply:
I am not aware of any horizontal-only models, but point taken and will change accordingly.